update-docker-repo-info.py

```diff
@@ -92,6 +104,34 @@ def get_os_release(image_name):
     return res.stdout.splitlines()
 
 
+def get_python_version(docker_info: str) -> str:
+    return python_version.group(1) if (
+        python_version := re.search(r'PYTHON_VERSION=(\d+\.\d+\.\d+)', docker_info)
+    ) else ''
+
+
+def add_python_version_for_dockerfiles_metadata(image_name: str, docker_info: str):
+    try:
+        if match := re.match(DOCKER_IMAGE_REGEX_PATTERN, image_name):
```

Comment:
is there a legitimate case why this would fail?

Reply:
its better to handle the case that it fails so we would know there is a problem in the future, i don't want to assume that every input is valid.